content/v2/openapi.yml

```
@@ -1838,6 +1856,57 @@ paths:
               schema:
                 $ref: '#/components/schemas/Error'
       summary: Check zone record distribution
+  '/{account}/zones/{zone}/resolution':
+    put:
+      description: |-
+        Activate DNS services for the zone.
+
+        Under Solo and Teams plans, active zones are charged when renewing your subscription to
+        DNSimple
+      parameters:
+        - $ref: '#/components/parameters/Account'
+        - $ref: '#/components/parameters/Zone'
+      operationId: enableZoneResolution
+      tags:
+        - zones
+      responses:
+        '200':
+          description: Successful activation of DNS services on the zone.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    $ref: '#/components/schemas/Zone'
+        '404':
+          $ref: '#/components/responses/404'
+      summary: Enable a zone's DNS resolution
```

Still references enable/disable.

```suggestion
      summary: Activate a zone DNS service
```